students/revyakinpetr/2/test_cats_composition.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+import glob
+import os
+import unittest
+
+import cats_composition
+
+
+class TestCatsComposition(unittest.TestCase):
+    """Unittest class for cats composition."""
+
+    def test_main(self):
+        """Test main function."""
+        test_index = 1
+
+        cat_processor = cats_composition.CatProcessor(
```

Comment:
The idea of this task was that it is possible to pass custom implementation of all handlers to `CatProcessor`. This test just tests the default implementation. Which is ok, but is not enough.